armada_jupyter/armada.py

```diff
@@ -39,13 +41,63 @@ def submit(submission: Submission):
             channel_credentials,
         )
 
-    client = ArmadaClient(channel)
+    client = parent_client(channel)
```

Comment:
Shouldn't we treat the client as an immutable object?  If a client is initialized with a channel, why would you override it with your own settings?

Reply:
I did this to allow for testing. We want to make sure we can pass a different client if needed.